integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/EnvironmentDatalakeClusterTest.java

```diff
@@ -108,14 +118,18 @@ public void testCreateDalalakeDeleteFails(TestContext testContext) {
                 .withInstanceGroupsEntity(setInstanceGroup(testContext))
                 .withCluster(setResources(testContext, rdsList,  testContext.get(LdapConfigTestDto.class).getName(), null, BP_NAME_DL))
                 .when(Stack.postV4())
-                .deleteGiven(LdapConfigTestDto.class, ldapConfigTestClient.delete(), key(FORBIDDEN_KEY))
-                .deleteGiven(DatabaseEntity.class, DatabaseEntity::delete, key(FORBIDDEN_KEY))
-                .deleteGiven(CredentialTestDto.class, Credential::delete, key(FORBIDDEN_KEY))
-                .deleteGiven(EnvironmentEntity.class, Environment::delete, key(FORBIDDEN_KEY))
+                .deleteGiven(LdapConfigTestDto.class, ldapConfigTestClient.delete(), key(forbiddenKey))
+                .deleteGiven(DatabaseEntity.class, DatabaseEntity::delete, key(forbiddenKey))
+                .deleteGiven(CredentialTestDto.class, Credential::delete, key(forbiddenKey))
+                .deleteGiven(EnvironmentEntity.class, Environment::delete, key(forbiddenKey))
```

Comment:
keys are unnecessary

Reply:
This is how delete given works. I don't want to change that part of the code.